beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -179,6 +180,13 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
         Ok(())
     }
 
+    /// Attempt to persist the enrs in the DHT to `self.store`.
+    pub fn persist_dht(&self, enrs: PersistedDht) -> Result<(), Error> {
```

Comment:
I don't think this should be on the `BeaconChain`. Presently the `BeaconChain` has no idea that the network exists and it would be nice to keep this separation.

Reply:
Thanks @paulhauner. Wanted to keep the separation but the only existing `store` was in the `BeaconChain`. Would you recommend having a separate `store` for the `NetworkService`?